compose/release-notes.md

```diff
@@ -7,6 +7,69 @@ redirect_from:
   - /release-notes/docker-compose/
 ---
 
+## 1.28.0 
+(2021-01-20)
+
+### Features
+
+- Added support for Nvidia GPUs via device requests.
+
+- Added support for service profiles.
+
+- Changed the SSH connection approach to the Docker CLI's via shellout to the local SSH client (old behaviour enabled by setting `COMPOSE_PARAMIKO_SSH` environment variable).
```

Comment:
```suggestion
- Changed the SSH connection approach to the Docker CLI by shelling out to the local SSH client. You can enable the old behaviour by setting `COMPOSE_PARAMIKO_SSH` environment variable.
```

Reply:
@aiordache Let me know if this makes sense?

Reply:
Does the `COMPOSE_PARAMIKO_SSH` need to contain a specific value (e.g. `COMPOSE_PARAMIKO_SSH=1`)?

Perhaps use active voice for the second sentence;

```suggestion
- Changed the SSH connection approach to the Docker CLI by shelling out to the local SSH client. Set the `COMPOSE_PARAMIKO_SSH` environment variable to enable the old behavior.
```